### PR TITLE
Support all version control systems that pip does

### PR DESCRIPTION
**Pull request by vilhalmer**
Ran into needing SVN immediately upon trying this out, so I figured I'd finish that TODO for you. :)

### CODE REVIEW

File: pipenv/utils.py [hunk 1]
```diff
@@ -83,23 +83,26 @@ def convert_deps_to_pip(deps, r=True):
...
-            else:
-                dep = ''
+        # Support for version control
+        maybe_vcs = [vcs for vcs in ('git', 'svn', 'hg', 'bzr') if vcs in deps[dep]]
+        vcs = maybe_vcs[0] if maybe_vcs else None
+
+        if vcs:
+            extra = '{0}+{1}'.format(vcs, deps[dep][vcs])
+
+            if extra:
```
Comment (nateprewitt) [inline pipenv/utils.py]: Is there a reason for adding this conditional? Given `extra = '{0}+{1}'.format(vcs, deps[dep][vcs])`, when can `extra` be a non-truthy value?

**Comment by kennethreitz** [comment]
looks good!

**Comment by nateprewitt** [comment]
Thanks for the work @vilhalmer!

**Comment by vilhalmer** [comment]
Glad to be of help!

**Pull request merged**